.env.example

```
@@ -9,4 +9,8 @@ ROOT=FALSE
 SYNAPSE_TOKEN=yourtoken
 
 # Port on the host that connects to RStudio
-HOST_PORT=80
\ No newline at end of file
+HOST_PORT=80
+
+# User-specified settings made visible to RStudio
+APP_FOO=10
+APP_BAR=hello
```

```suggestion
APP_BAR=hello

```
Suggestion to add a final line ending to this. Not needed, but I find that some apps can struggle with files if there's no final line ending.